### DISCUSSION TITLE
Dropping support for DropDown flyout in the future release

### DISCUSSION THREAD

**Discussion by ALTaleX531** [ALTaleX531/TranslucentFlyouts#85] [Announcements]
I've noticed that not many people will notice this flyout, but it makes the project difficult to maintain, and as it has been completely removed in newer versions of Windows 11, I intend to remove support for it in a future release.

**Comment by ghost** [comment]
Windows were going to make the context menu better or even usable but they always fail to deliver on their promise.
I am glad that you came up with this feature to make the old flyout look cool without using any extra resource as well.
I hope you'll eventually get more support developing this but I personally have been always satisfied with this work.
Upgraded to windows 11 22h2 and currently running 23h2 with latest TF
This is 23h2 patched with UltraUXThemePatcher and running Nivuu's theme and TF on top
![Screenshot 2024-01-06 080320](https://github.com/ALTaleX531/TranslucentFlyouts/assets/100816109/01406504-1dd3-43ad-b153-68c59fe521d5)

**Comment by ALTaleX531** [comment]
Good news, the DropDown flyout will be preserved in TranslucentFlyouts v3 and will be able to use the fluent animation. BTW, TranslucentFlyouts v3 fixes all the bugs and conflicting issues, extends the functionality and optimizes the stability!
